src/main/java/uk/gov/dvsa/mot/fixtures/generic/WebStepDefinitions.java

```diff
@@ -33,8 +33,8 @@ public WebStepDefinitions(WebDriverWrapper driverWrapper) {
         When("^I press the \"([^\"]+)\" button$", (String buttonText) ->
                 driverWrapper.pressButton(buttonText));
 
-        When("^I press the first \\{([^\\}]+)\\} button$", (String buttonText) ->
-                driverWrapper.pressFirstButton(driverWrapper.getData(buttonText)));
+        When("^I press the first \"([^\"]+)\" button$", (String buttonText) ->
```

Comment:
Changing this step will cause other tests to fail. Is there a reason why it changed?

Reply:
My bad. I was unable to find any usages of this step. I can now see that it's used somewhere else. I will revert this change.

Reply:
I created a new step definition, the one mentioned above should remain intact.